resolwe_bio/processes/support_processors/archive.yml

```diff
@@ -107,18 +121,29 @@
         re-checkrc "Crating IGV session failed"
       done
 
-      expressionmerge_archive.py \
-        --file_path "${FILE_PATH[@]}" \
-        --sample_names "${SAMPLE_NAMES[@]}" \
-        --build "${BUILD[@]}" \
-        --species "${SPECIES[@]}" \
-        --exp_type "${EXP_TYPE[@]}"
+      if [ $(echo -n "${EXP_TYPE[@]}" | wc -c) -ne 0 ]; then
```

Comment:
Length of the list can be checked by `"${#FILE_PATH[@]}"` as recently accepted to the current master branch